src/Objects/User.php

```diff
@@ -14,6 +20,57 @@
 class User extends BaseObject
 {
     /**
+     * @var string $currentConversation
+     */
+    private $currentConversation;
+
+    /**
+     * @var SQLite3Cache $cache
+     */
+    private $cache;
+
+    /**
+     * @inheritdoc
+     */
+    public function __construct($data)
+    {
+        parent::__construct($data);
+
+        $this->cache = new FilesystemCache(__DIR__.'/../Storage/cache');
```

Comment:
I think we could've used `illuminate/cache` and support multiple cache drivers?

For example, I prefer redis or memcached over a file cache.